<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_KbdControlMap</class>
 <widget class="QDialog" name="Dialog_KbdControlMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Keyboard Controls Mapping</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/music.png</normaloff>images/music.png</iconset>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#0055ff;&quot;&gt;Use this dialog to map a MIDI controller keyboard knobs, sliders and pusbuttons MIDI control codes to the Modular Synthesizer knobs,sliders, pushbuttons and other function.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 238, 102);background-color: rgb(40, 40, 40);</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox_dialogKbdControlSequences">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>200</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_KbdKontrolMapMapping">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>646</width>
     <height>156</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Mapping</string>
   </property>
   <widget class="QGroupBox" name="groupBox_midiControlSequenceData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>216</width>
      <height>86</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Midi Control Sequence</string>
    </property>
    <widget class="QTextEdit" name="textEdit_midiControlSequenceData_ch">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>50</y>
       <width>56</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_midiControlSequenceData_ch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>66</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_midiControlSequenceData_controlNum">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>56</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_midiControlSequenceData_ch_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>25</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Control</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_midiControlSequenceData_val">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>50</y>
       <width>56</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_midiControlSequenceData_val">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>25</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Value</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_synthControlData">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>35</y>
      <width>401</width>
      <height>86</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Synth. Control Data</string>
    </property>
    <widget class="QTextEdit" name="textEdit_synthControlData_module">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>50</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_synthControlData_module">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>25</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Module</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_synthControlData_controlId">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>50</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_synthControlData_controlId">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>25</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Control</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_synthControlData_val">
     <property name="geometry">
      <rect>
       <x>335</x>
       <y>50</y>
       <width>56</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_synthControlData_val">
     <property name="geometry">
      <rect>
       <x>335</x>
       <y>25</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Value</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_kbdControlSequncesTrain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>125</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Train</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_kbdControlSequncesMap">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>125</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Map</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_synthControlData_lastMapFile">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>125</y>
      <width>301</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_synthControlData_module_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>125</y>
      <width>96</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Last MAP File</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_kbdControlMapFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>File</string>
   </property>
   <widget class="QPushButton" name="pushButton_kbdControlFileOpen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_kbdControlFileSave">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox_synthControlMap_ignoreChannel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>186</y>
     <width>126</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Ignore Channel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_dialogKbdControlSequences</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_KbdControlMap</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_dialogKbdControlSequences</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_KbdControlMap</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
